roles/addons/diskless/files/disklessset.py

```diff
@@ -356,7 +398,13 @@ def generate_ipxe_boot_file(image_type, image_name, image_kernel, image_initramf
             elif selected_livenet_type == '1':
                 os.system('dnf groupinstall -y "core"  --setopt=module_platform_id=platform:el8 --installroot=/mnt')
             elif selected_livenet_type == '4':
-                os.system('dnf install -y @core {0} --setopt=module_platform_id=platform:el8 --installroot=/mnt'.format(selected_packages_list))
+                try:
+                    if os.system('dnf install -y @core {0} --setopt=module_platform_id=platform:el8 --installroot=/mnt'.format(selected_packages_list)) != 0:
+                        raise Exception('dnf does not exist')
+                except:
+                    print(bcolors.FAIL+'[ERROR] dnf: a package was not found or the repositories are break!'+bcolors.ENDC)
```

Comment:
dnf: a package was not found or the repositories are **broken**!

Reply:
Done.